docs/clients/p4rt-ctl.rst

```diff
@@ -116,15 +116,15 @@ Details about the supported commands
 
     $ p4rt-ctl set-pipe SWITCH PROGRAM P4INFO
     $ Example: p4rt-ctl set-pipe br0 <path>/sample.pb.bin <path>/p4info.txt
+ ..
 
-  .. note::
-
-    ``SWITCH``: Refers to the bridge name, which maps to device name
-    internally.
-    ``PROGRAM``: Refers to protobuf binary file generated by
-    tdi_pipeline_builder executable and the P4 program's configuration file.
-    ``P4INFO``: Refers to P4Info.txt file generated by p4c compiler
-    for a P4 program.
+   Parameter description:
```

Comment:
This section looks awful when rendered. I'd go back to the original version and try again some other time.

Reply:
Done